RetailCoder.VBE/Refactorings/EncapsulateField/EncapsulateFieldPresenterFactory.cs

```diff
@@ -7,10 +8,10 @@ namespace Rubberduck.Refactorings.EncapsulateField
     public class EncapsulateFieldPresenterFactory : IRefactoringPresenterFactory<EncapsulateFieldPresenter>
```

Comment:
Not a mergestopper, just keeping a note that we have quite a few concrete factory implementations - that would be part of the `RubberduckModule` technical debt IMO.